test/e2e/network/service.go

```diff
@@ -484,6 +484,9 @@ var _ = SIGDescribe("Services", func() {
 	It("should be able to change the type and ports of a service [Slow]", func() {
 		// requires cloud load-balancer support
 		framework.SkipUnlessProviderIs("gce", "gke", "aws")
+		if framework.ProviderIs("gke", "gce") {
```

Comment:
Thanks! Probably worth to have a TODO comment?

Reply:
Done. Copied over the TODO from my previous PR.